config-template.yml

```diff
@@ -8,25 +8,29 @@ app_name: My awesome goodmap application
 
 ## -- Languages supported by application
 ## -- First language would be used as default
-languages:
-  - en
-#  - pl
+LANGUAGES:
+  en:
+    name: English
+    flag: uk
+  pl:
+    name: polski
```

Comment:
Are the names displayed somewhere? I don't know exactly but I think it was used inside of language select input so I would make by the same convention - starting with capital letter

Reply:
This is probably part of a bigger issue - splitting configuration into two:  `config.yml` and database part.